src/components/scenes/ChangeMiningFeeScene.js

```diff
@@ -196,21 +216,37 @@ const rawStyles = {
     justifyContent: 'space-between'
   },
 
+  row: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    justifyContent: 'flex-start'
+  },
+  paddingRightIcon: {
+    paddingLeft: '5%'
+  },
+
   saveButton: {
     marginTop: THEME.rem(1.25)
   }
 }
 const styles: typeof rawStyles = StyleSheet.create(rawStyles)
 
 export const ChangeMiningFeeScene = connect(
-  (state: RootState): StateProps => ({
-    networkFeeOption: getGuiMakeSpendInfo(state).networkFeeOption,
-    customNetworkFee: getGuiMakeSpendInfo(state).customNetworkFee,
-    spendTargets: getGuiMakeSpendInfo(state).spendTargets
-  }),
+  (state: RootState): StateProps => {
+    return {
+      networkFeeOption: getGuiMakeSpendInfo(state).networkFeeOption,
+      customNetworkFee: getGuiMakeSpendInfo(state).customNetworkFee,
+      currencySettings: state.ui.settings[getSelectedCurrencyCode(state)],
+      spendTargets: getGuiMakeSpendInfo(state).spendTargets
+    }
+  },
   (dispatch: Dispatch): DispatchProps => ({
-    onSubmit(networkFeeOption: string, customNetworkFee: Object, walletId: string, currencyCode?: string) {
-      dispatch(sendConfirmationUpdateTx({ networkFeeOption, customNetworkFee }, true, walletId, currencyCode))
+    // dispatch = setter
+    onSubmit(networkFeeOption: FeeOption, isDefault: boolean, customFee: JsonObject, walletId: string, currencyCode?: string) {
+      if (isDefault && currencyCode) {
```

Comment:
There's a bug here which prevents ever turning off the default fee option.

Reply:
Fixed and pushed with the setting of default fee to 'none' and checking for that in the case the user toggles the default fee off.